tcmalloc/internal/percpu.h

```diff
@@ -19,7 +19,8 @@
 
 // TCMALLOC_PERCPU_RSEQ_SUPPORTED_PLATFORM defines whether or not we have an
 // implementation for the target OS and architecture.
-#if defined(__linux__) && (defined(__x86_64__) || defined(__PPC64__))
+#if defined(__linux__) \
```

Comment:
Small nit:  When I look at pulling this in, clang-format wants to rewrite this as:

```
#if defined(__linux__) && \
    (defined(__x86_64__) || defined(__PPC64__) || defined(__aarch64__))
```

Reply:
Done.